### PR TITLE
openssl@1.1 1.1.1g

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by claui — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           mkdir bottles
           cd bottles
-          brew test-bot
+          brew test-bot --skip-recursive-dependents
```
Comment (alex) [inline .github/workflows/tests.yml:36]: Is this PR mergable with this change? I know without it, testing OpenSSL tests an extremely long time.

Comment (Bo98) [reply .github/workflows/tests.yml:36]: It is mergable but this commit will not be included in that merge. I put it in a separate commit for that reason.

Comment (Bo98) [reply .github/workflows/tests.yml:36]: In the future we might have something better - but this is what we thought of over the weekend in anticipation of the OpenSSL update today.

Comment (alex) [reply .github/workflows/tests.yml:36]: Thanks -- we (pyca/cryptography) redistribute a statically linked copy of OpenSSL from brew, so I'm very appreciative that y'all were planning ahead for today!

**Comment by Bo98** [comment]
Merged: 6a23e8c6997dfe3f42055218fa884e07877f9948 (accidentally removed the close message).

**Pull request closed without merging**